service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -600,6 +614,80 @@ void shouldNotifyAdminOfPlacementApplicationUpload() throws Exception {
         );
     }
 
+    @Test
+    void shouldSendNotificationToPartiesWhenAddedToCaseByEmail() throws IOException, NotificationClientException {
+        final Map<String, Object> expectedParameters = ImmutableMap.<String, Object>builder()
+            .put("firstRespondentLastName", "Moley")
+            .put("familyManCaseNumber", "123")
+            .put("reference", "12345")
+            .build();
+
+        List<Element<Representative>> representatives = new ArrayList<>();
+        Element<Representative> representative = Element.<Representative>builder()
+            .value(Representative.builder()
+                .email("joe-blogs@gmail.com")
+                .servingPreferences(EMAIL).build()).build();
+        representatives.add(representative);
+
+        RepresentativeServingPreferences preferences = EMAIL;
```

Comment:
This can be inlined

Reply:
This was just being pushed up so resolving